sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-disbursements-without-valid-supplier-report.sql

```diff
@@ -0,0 +1,63 @@
+INSERT INTO
+  sims.report_configs (report_name, report_sql)
+VALUES
+  (
+    'Disbursements_Without_Valid_Supplier_Report',
+    'WITH disbursement_sums AS (
+	    SELECT
+	        students.id AS student_id,
+	        SUM(CASE WHEN dv.value_code = ''BCAG'' THEN dv.effective_amount ELSE 0 END) AS bcag_effective_amount,
+	        SUM(CASE WHEN dv.value_code = ''SBSD'' THEN dv.effective_amount ELSE 0 END) AS sbsd_effective_amount,
+	        SUM(CASE WHEN dv.value_code = ''BGPD'' THEN dv.effective_amount ELSE 0 END) AS bgpd_effective_amount
+	    FROM
```

Comment:
The conditions applied to the JOIN are usually applied to the WHERE clause unless required in the join. Was there a reason to restrict the query in the JOIN instead aof dding the filter in the WHERE?